static/templates/settings/uploaded_files_list.hbs

```diff
@@ -17,7 +17,7 @@
         </div>
         {{/if}}
     </td>
-    <td>{{ size_str }}</td>
+    <td class="drop_size_str" >{{ size_str }}</td>
```

Comment:
This class name can be improved. Not sure why you added "drop" here. Maybe using `upload-size` itself is fine as there is no other CSS for `upload-size` class.

Reply:
`upload-size` class is already being used by the file size column, so changed it to upload-size-str.

Reply:
There is no problem in using same class, since it is used in the heading of same column only and both are used to apply same CSS.